README.md

```diff
@@ -1,2 +1,54 @@
 # server-plugin-ore
 Generates ore veins
+
+## Add your own ore
+Other plugins can register ore blocks with this plugin by extending its configuration:
+´´´
+konstructs {
+  ...
+  org/konstructs/ore.ores {
+    // The config key is the block type of the ore, i.e.
+    // the block type of the ore is here my/domain/my-ore
+    my/domain/my-ore {
+      // This is the block type in which the ore can spawn
+      spawns-in = org/konstructs/stone
```

Comment:
Maybe leave this for a future improvement, but I see several cases where I prefer this to be a list of several candidates.